src/app/excel/excel-table/excel-table.component.html

```diff
@@ -47,71 +67,44 @@ <h5><span class="bold">{{sprintStartDate | date: 'dd-MM-yyy'}}</span> -
         <hr>
       </div>
     </div>
-    <div id="availability-summary">
-      <div class="ui-g">
-        <div class="ui-g-2">
-          <h5>Suma godzin</h5>
-        </div>
-        <div class="ui-g-1">
-          <span>{{hoursSum}}</span>
-        </div>
-      </div>
-      <div class="ui-g">
-        <div class="ui-g-2">
-          <h5>Aktualny współczynnik</h5>
-        </div>
-        <div class="ui-g-1">
-          <span>{{actualRate}}</span>
-        </div>
-      </div>
-    </div>
 
-    <div class="ui-g-12">
-      <p>L20: {{declarableSprint.estimatedTimePlanned}}</p>
-      <p>L21: {{declarableSprint.finalTimePlanned}}</p>
-      <p>L22: {{declarableSprint.timeBurned}}</p>
-    </div>
+    <app-availability-summary [currentSprint]="currentSprint"
+                              [sprintSummary]="sprintSummary"></app-availability-summary>
 
-    <p-button *ngIf="declarableSprint.sprintState === 'FINISHED'"
+    <p-button *ngIf="currentSprint.sprintState === 'FINISHED'"
               label="Zamknij sprint" icon="pi pi-check"
               (click)="closeSprint()"
               class="mr-2">
     </p-button>
 
-    <p-button *ngIf="declarableSprint.sprintState !== 'UPCOMING'"
+    <p-button *ngIf="currentSprint.sprintState !== 'UPCOMING'"
               label="Synchronizuj z Jirą" icon="pi pi-refresh"
               (click)="synchronizeWithJira()"
               class="mb-2">
     </p-button>
 
-    <ng-template [ngIf]="isSynchronized">
+    <ng-template [ngIf]="showSprintSelectMenu">
       <h6>Dostępne sprinty:</h6>
       <p-table [value]="jiraSprints">
         <ng-template pTemplate="header">
           <tr>
             <th>Nazwa</th>
             <th>Data rozpoczęcia</th>
             <th>Stan</th>
-            <th></th>
           </tr>
         </ng-template>
         <ng-template pTemplate="body" let-jiraSprint>
-          <tr>
+          <tr class="highlight-on-hover-content clickable" (click)="chooseJiraSprint(jiraSprint)">
             <td>{{jiraSprint.name}}</td>
             <td>{{jiraSprint.startDate}}</td>
             <td>{{jiraSprint.state}}</td>
-            <td>
-              <p-button icon="pi pi-arrow-right"
-                        (onClick)="chooseJiraSprint(jiraSprint)">
-              </p-button>
-            </td>
           </tr>
         </ng-template>
       </p-table>
     </ng-template>
 
   </ng-template>
-  <ng-template [ngIf]="isLoaded && !declarableSprint">
+  <ng-template [ngIf]="isSprintLoaded && !currentSprint">
```

Comment:
Ten warunek chyba nie działa. Gdy nie mam żadnego sprintu w projekcie, to nie dostaję informacji zawartej w divie poniżej.

Reply:
A tak poza tym, to bardzo ładnie :)

Reply:
jednak te observable mnie nie przestaną zaskakiwać